### ISSUE TITLE
Plutonium Specific Fixes: Origins 8 Player Bugs: Intersection tracker causes players to die on spawn occasionally

### ISSUE THREAD

**Issue by JezuzLizard** [JezuzLizard/Recompilable-gscs-for-BO2-zombies-and-multiplayer#26]
The intersection tracker will be disabled since its benefit is very low.

**Comment by INSANEMODE** [comment]
seeing everyone go down together from intersection on spawn is a headache, but i its pretty fun to see when it happens the first time.

**Comment by JezuzLizard** [comment]
Yeah I've had a few games end because of the intersection tracker. On Die Rise I've had games where all 8 players died on spawn after a game ended mutliple times its a meme. Origins is generally the same too.

**Comment by INSANEMODE** [comment]
the quick fix that i use on my servers is pretty much just taking level.player_intersection_tracker_override and make it always return 1.

level.player_intersection_tracker_override = ::clientids_player_intersection_tracker_override

clientids_player_intersection_tracker_override()
{
	return 1;
}

works on die rise, origins, and motd, from what i've tested so far.  have had no issues.
